### PR TITLE
Fix links 

### PR DESCRIPTION
**Pull request by amarantha-k**

### CODE REVIEW

File: docs/concepts/tokens/decentralized-exchange/automated-market-makers.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ labels:
 ---
 # What Is an Automated Market Maker (AMM)?
 
-_(Requires the [AMM amendment][] {% not-enabled /%})_
+_(Requires the [XLS-30 AMM amendment][] {% not-enabled /%})_
```
Comment (DennisDawson) [inline docs/concepts/tokens/decentralized-exchange/automated-market-makers.md]: This doesn't render in the preview.

**Review by DennisDawson — APPROVED** [review]
The amendment link in Automated Market Makers doesn't render in preview, not sure if it will render in production. The semantics table renders properly, but in English, so I'm not sure that makes sense. The other links appear to work (EN and JA).

File: docs/concepts/tokens/decentralized-exchange/automated-market-makers.md (hunk 1 above)
Comment (amarantha-k) [reply docs/concepts/tokens/decentralized-exchange/automated-market-makers.md]: Good catch! Will fix this.

**Comment by amarantha-k** [comment]
Tagging @tequdev to translate the depositauth semantics table (docs/_snippets/depositauth-semantics-table.md)

**Pull request merged**